frontend/src/lib/format.ts

```diff
@@ -62,6 +76,9 @@ class Format {
  * Formats the string nicely if it's a floating point, number, date or duration.
  */
 function toFormattedString(x: any): string {
+  if (moment.isMoment(x)) {
+    return x.format()
```

Comment:
Not 100% from what I read on the docs but wouldn't this just end up parsing the time back into local timezone?

Reply:
This is formatting, not parsing — `x` itself is already a moment object, we're just rendering it.